components/library/zlib/Makefile

```diff
@@ -57,7 +57,7 @@ gcc_OPT="-O2"
 
 # Also, the x86 architecture does not require alignment for multi-byte
 # loads, so we can define UNALIGNED_OK for x86
-CFLAGS.32 += -DUNALIGNED_OK
+CFLAGS.i386.32 += -DUNALIGNED_OK
```

Comment:
I'm not sure this will work as expected - we don't have CFLAGS+=$(CFLAGS.$(MACH).$(BITS)) anywhere